configuration/concepts/commons.csv

```diff
@@ -110,7 +110,7 @@ f891f12b-9947-4cc4-aa97-b6ba47567a58,,CIEL:162141,Allergies,Allergies,Allergies,
 b5f7f595-88ea-4522-ba92-96454240a54f,,,Existent,Existant,Existent,Existant,,,Misc,N/A,,,,,,,,,,,
 b821d08c-34f3-4614-9ae6-30a80bce0236,,CIEL:1499,Moderate,Modéré,Moderate,Modéré,,,Misc,N/A,,,,,,,,,,,
 c728a231-e76a-4c68-aba6-a3909640f5b3,,CIEL:1500,Severe,Sévère,Severe,Sévère,,,Misc,N/A,,,,,,,,,,,
-ba9c0c84-3f0b-47cf-8cee-d03e84775688,,,Consultation Note,Note de consultation,Consultation Note,Note de consultation,,,Misc,Text,,,,,,,,,,,
+ba9c0c84-3f0b-47cf-8cee-d03e84775688,,,Consultation Note,Consultation Note,Consultation Note,Consultation Note,,,Misc,Text,,,,,,,,,,,
```

Comment:
In fr: "Note de consultation"

Reply:
the UI breaks....It appears bahmniapps creates a rest call using a specific name while agnostic of the default locale

Reply:
Understood.